src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```diff
@@ -27,6 +27,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and '$(TargetPlatformVersion)' == ''" >
     <TargetPlatformVersion>7.0</TargetPlatformVersion>
     <!-- Automatic generation of TargetPlatformMoniker and TargetPlatformDisplayName have already been evaluated so we need to hardcode these values. -->
+    <SupportedOSPlatform>7.0</SupportedOSPlatform>
```

Comment:
Is that the right name? I thought [we agreed](https://github.com/dotnet/designs/pull/157) on `SupportedOSPlatformVersion`.

/cc @dsplaisted

Reply:
It looks like it got renamed to `SupportedOSPlatform` [here](https://github.com/dotnet/sdk/pull/12775/files#diff-b1cca2205c228925d515de229057c7c6R81).

Reply:
We decided to rename it to SupportedOSPlatformVersion, but haven't done so yet.  I was planning to do that as part of #13453